packages/toolpad-app/src/runtime/evalJsBindings.ts

```diff
@@ -252,6 +255,13 @@ export default function evalJsBindings(
           const nestedBindingResult = results[nestedBindingId];
 
           if (nestedBindingResult) {
+            if (!nestedBindingsError) {
```

Comment:
Would be interesting to add a test for the error/loading propagation (There are some examples of these in the `backend-basic` test) for when we start overhauling the binding engine.

Reply:
I'll give it a try, if it's too much I'll split it into a separate PR.

Reply:
Working on it separately just to keep this simpler.